internal/kafka/internal/environments/development.go

```diff
@@ -36,5 +36,6 @@ func NewDevelopmentEnvLoader() environments.EnvLoader {
 		"observability-red-hat-sso-tenant":                "managedkafka",
 		"observatorium-auth-type":                         "dex",
 		"max-allowed-developer-instances":                 "1",
+		"admin-authz-config-file":                         "config/admin-authz-configuration.yaml",
```

Comment:
```suggestion
```
not needed as it is set via https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/fd1700778497d649ae4943d8d732c664f6361e28/pkg/auth/roles_authz.go#L38